templates/pricing/index.html

```diff
@@ -29,7 +29,8 @@ <h2 class="p-heading--three">Open Infrastructure support and security</h2>
           <li class="p-list__item is-ticked">SDN</li>
           <li class="p-list__item is-ticked">Storage</li>
         </ul>
-        <p><a href="/pricing/infra/">See our services&nbsp;&rsaquo;</a>
+        <p><a class="p-button--neutral" href="/pricing/infra/">See our services</a></p>
+        <p><a class="p-link--external" href="https://buy.ubuntu.com/">Purchase from the store</a>
```

Comment:
Personally I would switch these by making the See our services a link and the Purchase from the store a positive button.